### PR TITLE
references.md sort

### PR DESCRIPTION
**Pull request by vitaly-zdanevich**
Because looks like mostly it was already sorted.

### CODE REVIEW

File: templates/shortcodes/references.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
+[FOSStodon]: https://fosstodon.org/@organicmaps "FOSStodon"
+[LinkedIn]:  https://www.linkedin.com/company/organic-maps/
```
Comment (biodranik) [inline templates/shortcodes/references.md:2]: Let's apply collation sorting here, it is more natural for humans. This page is for humans )

Comment (biodranik) [reply templates/shortcodes/references.md:2]: https://learn.microsoft.com/en-us/globalization/locale/sorting-and-string-comparison

Comment (vitaly-zdanevich) [reply templates/shortcodes/references.md:2]: How to do that?

Comment (biodranik) [reply templates/shortcodes/references.md:2]: https://icu4c-demos.unicode.org/icu-bin/collation.html
(select alternate -> shifted to change order with underscores)
```
 [appgallery]: https://appgallery.huawei.com/#/app/C104325611?local={{ lang }} "{{ trans(key='install-appgallery', lang=lang) }}"
[appgallery_review]: appmarket://details?id=app.organicmaps
[appstore]: https://apps.apple.com/app/organic-maps/id1567437057 "{{ trans(key='install-appstore', lang=lang) }}"
[appstore_review]: https://itunes.apple.com/app/id1567437057?action=write-review
[contributing]: https://github.com/organicmaps/organicmaps/blob/master/docs/CONTRIBUTING.md
[email]: mailto:support@organicmaps.app
[exodus]: https://reports.exodus-privacy.eu.org/en/reports/app.organicmaps/latest/
[facebook]: https://facebook.com/OrganicMaps
[fdroid]: https://f-droid.org/{{ lang | truncate(length=2, end="") }}/packages/app.organicmaps/ "{{ trans(key='install-fdroid', lang=lang) }}"
[firebase]: https://appdistribution.firebase.dev/i/2f0fee463107b137
[flatpak]: https://flathub.org/apps/details/app.organicmaps.desktop
[FOSStodon]: https://fosstodon.org/@organicmaps "FOSStodon"
[github]: https://github.com/organicmaps/organicmaps
[github issues]: https://github.com/organicmaps/organicmaps/issues "GitHub Issues"
[googleplay]: https://play.google.com/store/apps/details?id=app.organicmaps&hl={{ lang }} "{{ trans(key='install-googleplay', lang=lang) }}"
[googleplay_review]: market://details?id=app.organicmaps
[gpx_issue]: https://github.com/organicmaps/organicmaps/issues/624
[ideas]: https://github.com/organicmaps/organicmaps/discussions/categories/ideas
[instagram]: https://instagram.com/organicmaps.app/
[issues]: https://github.com/organicmaps/organicmaps/issues
[LinkedIn]: https://www.linkedin.com/company/organic-maps/
[mapswithme]: https://en.wikipedia.org/wiki/Maps.me "Maps.Me"
[mastodon]: https://mastodon.social/@organicmaps "Mastodon"
[matrix]: https://matrix.to/#/#organicmaps:matrix.org "Organic Maps on the Matrix"
[matrix_ru]: https://matrix.to/#/#organicmapsru:matrix.org "Organic Maps в Matrix"
[mythic_beasts_donation]: https://www.mythic-beasts.com/blog/2021/10/06/improving-the-world-bit-by-expensive-bit/
[mythic_beasts]: https://www.mythic-beasts.com/
[openstreetmap]: https://www.openstreetmap.org/about "OpenStreetMap"
[public_transport_validator]: https://cdn.organicmaps.app/subway/
[reddit]: https://www.reddit.com/r/organicmaps/
[repology]: https://repology.org/project/organicmaps/versions
[stripe_chf]: https://donate.organicmaps.app/?currency=chf "Donate in CHF"
[stripe_eur]: https://donate.organicmaps.app/?currency=eur "Donate in EUR"
[stripe_gbp]: https://donate.organicmaps.app/?currency=gbp "Donate in GBP"
[stripe]: https://donate.organicmaps.app/ "Donate"
[stripe_pln]: https://donate.organicmaps.app/?currency=pln "Przekaż złotówki"
[stripe_uah]: https://donate.organicmaps.app/?currency=uah "Пожертвуйте в гривнях"
[stripe_usd]: https://donate.organicmaps.app/?currency=usd "Donate in USD"
[telegram_chat]: https://t.me/OrganicMaps
[telegram_chat_ru]: https://t.me/OrganicMapsRu
[telegram]: https://t.me/OrganicMapsApp
[testflight]: https://testflight.apple.com/join/lrKCl08I
[trackercontrol]: https://ios.trackercontrol.org/analysis/app.organicmaps
[translations_appstore]: https://hosted.weblate.org/projects/organicmaps/apple-appstore-metadata/
[translations_googleplay]: https://hosted.weblate.org/projects/organicmaps/fdroid-metadata/
[translations]: https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md
[translations_website]: https://hosted.weblate.org/projects/organicmaps/website/
[twitter]: https://twitter.com/OrganicMapsApp

```

Comment (vitaly-zdanevich) [reply templates/shortcodes/references.md:2]: image

File: templates/shortcodes/references.md [hunk 2]
```diff
@@ -5,44 +5,44 @@
...
+[mythic_beasts]: https://www.mythic-beasts.com/
 [openstreetmap]: https://www.openstreetmap.org/about "OpenStreetMap"
 [public_transport_validator]: https://cdn.organicmaps.app/subway/
 [reddit]: https://www.reddit.com/r/organicmaps/
 [repology]: https://repology.org/project/organicmaps/versions
-[stripe]: https://donate.organicmaps.app/ "Donate"
 [stripe_chf]: https://donate.organicmaps.app/?currency=chf "Donate in CHF"
 [stripe_eur]: https://donate.organicmaps.app/?currency=eur "Donate in EUR"
 [stripe_gbp]: https://donate.organicmaps.app/?currency=gbp "Donate in GBP"
+[stripe]: https://donate.organicmaps.app/ "Donate"
```
Comment (vitaly-zdanevich) [inline templates/shortcodes/references.md:35]: @biodranik your collation sorting - but strange that [stripe] is in the middle

Comment (biodranik) [reply templates/shortcodes/references.md:35]: That looks like a bug in their algo/implementation.

Comment (biodranik) [reply templates/shortcodes/references.md:35]: For consistency, a shorter string should be at the top, not at the bottom (see other examples in this file).

File: templates/shortcodes/references.md [hunk 3]
```diff
@@ -5,44 +5,44 @@
...
 [stripe_pln]: https://donate.organicmaps.app/?currency=pln "Przekaż złotówki"
 [stripe_uah]: https://donate.organicmaps.app/?currency=uah "Пожертвуйте в гривнях"
 [stripe_usd]: https://donate.organicmaps.app/?currency=usd "Donate in USD"
-[telegram]: https://t.me/OrganicMapsApp
 [telegram_chat]: https://t.me/OrganicMaps
 [telegram_chat_ru]: https://t.me/OrganicMapsRu
+[telegram]: https://t.me/OrganicMapsApp
```
Comment (biodranik) [inline templates/shortcodes/references.md:41]: ditto

File: templates/shortcodes/references.md [hunk 4]
```diff
@@ -5,44 +5,44 @@
...
 [testflight]: https://testflight.apple.com/join/lrKCl08I
-[translations]: https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md
+[trackercontrol]: https://ios.trackercontrol.org/analysis/app.organicmaps
 [translations_appstore]: https://hosted.weblate.org/projects/organicmaps/apple-appstore-metadata/
 [translations_googleplay]: https://hosted.weblate.org/projects/organicmaps/fdroid-metadata/
+[translations]: https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md
```
Comment (biodranik) [inline templates/shortcodes/references.md:46]: ditto

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**